app/build.gradle

```diff
@@ -7,7 +7,7 @@ android {
 
     defaultConfig {
         applicationId "com.example.bookwormadventuresdeluxe2"
-        minSdkVersion 16
+        minSdkVersion 23
```

Comment:
What was this for?

Reply:
uhoh no clue if it will break anything. it might do the old lambdas and stuff

Reply:
This was probably left over stuff from when I was experimenting with different barcode scanners. I'll revert this to 16.